apps/src/aichat/views/aichatView.module.scss

```diff
@@ -55,7 +55,7 @@ $column-min-width: 200px;
 }
 
 .startOverButton {
-  margin-right: 5px;
+  text-transform: capitalize;
```

Comment:
Added this because the `aichatI18n.aria_startOver()` string is all lowercase and looked inconsistent in the tooltip.